docs/metrics/getting-started/README.md

```diff
@@ -87,9 +87,9 @@ the configuration for metrics like `Meter` names, readers, etc. and is highly
 ## OpenTelemetry .NET special note
 
 Metrics in OpenTelemetry .NET is a somewhat unique implementation of the
-OpenTelemetry project, as most of the Metrics API are incorporated directly
-into the .NET runtime itself. From a high level, what this means is that you
-can instrument your application by simply depending on
+OpenTelemetry project, as most of the Metrics API are made part of the [.NET
+runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```

Comment:
```suggestion
OpenTelemetry project, as most of the Metrics API are offered from the [.NET
runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```

?

Reply:
Maybe "built into"? I'm not good at English, maybe getting some suggestion from native English speakers?

Reply:
```suggestion
OpenTelemetry project, as most of the
[Metrics API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md)
is implemented by the [.NET
runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```